Examples/UIComponentExample/Examples/Irregular Layout/Components/CustomizeLayout.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CustomizeLayout.swift
+//  UIComponentExample
+//
+//  Created by y H on 2021/7/30.
+//  Copyright © 2021 Luke Zhao. All rights reserved.
+//
+
+import UIKit
+import UIComponent
+
+struct CustomizeLayout: Component, HorizontalLayoutProtocol {
```

Comment:
There is a `Absolute` layout builtin.

Reply:
`Absolute`  cannot get the `constraint` parameter
Need `constraint` calc reasonable size
```swift
VStack(spacing: 2) {
  Absolute(children: [], frames: []) // cannot get the constraint parameter
}.inset(h: 10)
```